.github/workflows/release.yml

```diff
@@ -18,6 +18,18 @@ jobs:
           - "centos9"
           - "microos"
           - "slemicro"
+        include:
+          - os: centos7
+            rpm-s3-path: centos/7
+          - os: centos8
+            rpm-s3-path: centos/8
+          - os: centos9
+            rpm-s3-path: centos/9
+          - os: microos
+            rpm-s3-path: microos
+          - os: slemicro
+            rpm-s3-path: slemicro
```

Comment:
You already have the os's listed in the matrix, are you sure you need to repeat them all here?